coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,18 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(
+    torch_fx_node: torch.fx.Node,
+    skip_ops: List[str] = [],
+) -> bool:
+    assert torch_fx_node.op == "call_function"
+    
+    torch_fx_node_target_name = torch_fx_node.target.__name__.lower()
```

Comment:
Would coreml backend prefer using `torch_fx_node.target` (the exact operator) or use the name string? The first one can be more accurate, and we can call it directly (like `torch_fx_node.target(a, b)`. The second one ideally should be unique too.

Reply:
First, I would like to make sure: how does the exact operator look like? If it is an aten op, what would happen for core aten or edge dialects?

I would prefer the name string, for backward compatibility reason:
1. We used name string in supporting `torch.jit.trace` model conversions
2. Now for exir, we are leveraging our existing op translations (since it's still the same aten ops), so we are tied to the name string implementation